server/eclipse-project/src/main/java/us/freeandfair/corla/model/ComparisonAudit.java

```diff
@@ -534,19 +529,46 @@ private BigDecimal computeOptimisticSamplesToAudit(final int twoUnder,
    * Signals that a sample has been audited. This ensures that estimates
    * are recalculated correctly and states are updated.
    *
-   * @param the_count The count of samples that have been audited simultaneously
+   * @param count The count of samples that have been audited simultaneously
    * (for duplicates).
    */
-  public void signalSampleAudited(final int the_count) {
+  public void signalSampleAudited(final int count) {
     my_estimated_recalculate_needed = true;
-    my_audited_sample_count = my_audited_sample_count + the_count;
+    my_audited_sample_count = my_audited_sample_count + count;
 
     if (my_audit_status != AuditStatus.ENDED &&
         my_audit_status != AuditStatus.NOT_AUDITABLE) {
       my_audit_status = AuditStatus.IN_PROGRESS;
     }
   }
 
+  /**
+   *
+   * Signals that a sample has been audited, if the CVR was selected for
+   * this audit and this audit is targeted (i.e., not for opportunistic
+   * benefits.)
+   *
+   * @param count The count of samples that have been audited simultaneously
+   * @param cvrID ID of the CVR being audited
+   */
+  public void signalSampleAudited(final int count, final Long cvrID) {
+    final boolean covered = isCovering(cvrID);
+    final boolean targeted = contestResult().getAuditReason().isTargeted();
+
+    if (targeted && !covered) {
+      LOGGER.debug("[signalSampleAudited: Targeted but not covered.]");
```

Comment:
If we want to leave this in, we'll almost certainly want `cvrID` for correlation purposes.

Reply:
oh, yes, clearly. 
_/me puts on 👓_.